### PR TITLE
Bugfixed/further cli functionality

### PR DESCRIPTION
**Pull request by AneequeZubair9**
Fixed
Descriptions for the CLI parameters:
--allow-amend: Whether the deployed VM image may be changed and deployed on the same URL
--use-latest: Whether to use latest version of the base docker image
--reproducible: Not Implemented 
--internet: Whether the VM should have internet connectivity
--aleph_api: Whether the VM needs access to Aleph messages API

### CODE REVIEW

File: src/aleph_client/asynchronous.py [hunk 1]
```diff
@@ -64,9 +64,9 @@ def get_fallback_session() -> ClientSession:
 
 
 async def ipfs_push(
-    content,
-    session: Optional[ClientSession] = None,
-    api_server: str = settings.API_HOST,
+        content,
+        session: Optional[ClientSession] = None,
+        api_server: str = settings.API_HOST,
```
Comment (MHHukiewitz) [inline src/aleph_client/asynchronous.py:69]: Please use https://black.readthedocs.io/en/stable/ instead for auto-formatting code.

File: src/aleph_client/asynchronous.py [hunk 2]
```diff
@@ -296,26 +296,31 @@ async def create_store(
...
-    subscriptions: Optional[List[Dict]] = None,
+        account: Account,
+        program_ref: str,
+        entrypoint: str,
+        runtime: str,
+        allow_amend: bool,
+        use_latest: bool,
+        reproducible: bool,
+        internet: bool,
+        aleph_api: bool,
```
Comment (MHHukiewitz) [inline src/aleph_client/asynchronous.py:307]: You could assign default values here the same as the hardcoded ones before:
```suggestion
        aleph_api: bool = True,
```

File: src/aleph_client/commands/message.py [hunk 3]
```diff
@@ -1,56 +1,51 @@
...
+        type: str = typer.Option("test", help="Text representing the message object type"),
+        ref: Optional[str] = typer.Option(None, help=help_strings.REF),
+        channel: str = typer.Option(settings.DEFAULT_CHANNEL, help=help_strings.CHANNEL),
+        private_key: Optional[str] = typer.Option(
+            settings.PRIVATE_KEY_STRING, help=help_strings.PRIVATE_KEY
+        ),
+        private_key_file: Optional[Path] = typer.Option(
+            settings.PRIVATE_KEY_FILE, help=help_strings.PRIVATE_KEY_FILE
+        ),
+        debug: bool = False,
```
Comment (MHHukiewitz) [inline src/aleph_client/commands/message.py:48]: Same goes for formatting here

File: src/aleph_client/commands/program.py [hunk 4]
```diff
@@ -56,35 +43,67 @@
...
+        entrypoint: str = typer.Argument(..., help="Your program entrypoint"),
+        channel: str = typer.Option(settings.DEFAULT_CHANNEL, help=help_strings.CHANNEL),
+        memory: int = typer.Option(
+            settings.DEFAULT_VM_MEMORY, help="Maximum memory allocation on vm in MiB"
+        ),
+        allow_amend: bool = typer.Option(False, help='Deployed VM image may be changed and deployed on the same URL'),
+        use_latest: bool = typer.Option(True, help='Use latest version of the base docker image'),
+        reproducible: bool = typer.Option(False, help=' (Not Implemented)--> Check output is same for other nodes'),
+        internet: bool = typer.Option(True, help='VM should have internet connectivity'),
+        aleph_api: bool = typer.Option(True, help='VM needs access to Aleph messages AP'),
```
Comment (MHHukiewitz) [inline src/aleph_client/commands/program.py:56]:
```suggestion
        aleph_api: bool = typer.Option(True, help='VM needs access to Aleph messages'),
```

File: tests/unit/test_asynchronous_get.py [hunk 5]
```diff
@@ -57,3 +58,7 @@ async def test_get_messages():
     assert len(messages) > 1
     assert messages[0].type
     assert messages[0].sender
+
+
+if __name__ == '__main __':
+    unittest.main()
```
Comment (MHHukiewitz) [inline tests/unit/test_asynchronous_get.py:64]: Please revert changes here, we don't use `unittest`

**Review by MHHukiewitz — CHANGES_REQUESTED** [review]
LGTM, minor changes required. Try to use https://black.readthedocs.io/en/stable/ to format all the modified files according to the style as the repository is formatted in

**Comment by MHHukiewitz** [comment]
Changes here need to be reimplemented in aleph-sdk-python. Closed.

**Comment by hoh** [comment]
> Changes here need to be reimplemented in aleph-sdk-python. Closed.

Everything related to the CLI should be implemented in `aleph-client`, `aleph-sdk-python` is limited to libraries.

**Comment by MHHukiewitz** [comment]
Superseded by https://github.com/aleph-im/aleph-sdk-python/pull/77 and a future PR including the new parameters in the CLI.

**Pull request closed without merging**